common/openapi.yml

```diff
@@ -145,6 +145,24 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/v1/datasets/{id}/preview-metadata:
```

Comment:
Any thought between `/api/v1/datasets/{id}/preview-metadata` and `/api/v1/datasets/{id}/preview` ? Even though data repo calls this `metadata`, I think this is actually data (as opposed to the catalog entries, which I would consider metadata).

Reply:
So, there's going to be a separate endpoint to `getPreviewTable`. I could call them `preview` and `preview-table`? But as a user, I wouldn't know what that meant.